portal/server/controllers/dashboard/facilities/facilities-filters-query.js

```diff
@@ -40,38 +43,55 @@ const dashboardFacilitiesFiltersQuery = (filters, user) => {
       const fieldName = Object.keys(filterObj)[0];
       const filterValue = filterObj[fieldName];
 
-      const isKeywordField = fieldName === CONTENT_STRINGS.DASHBOARD_FILTERS.BESPOKE_FIELD_NAMES.KEYWORD;
+      switch (fieldName) {
+        case BESPOKE_FIELD_NAMES.KEYWORD: {
```

Comment:
How about something like this instead, whilst we're here, easer to read/understand

```js
case BESPOKE_FIELD_NAMES.KEYWORD: {
  const keywordValue = filterValue[0];

  const keywordFilter = {
    OR: keywordQuery(keywordValue),
  };

  query.AND.push(keywordFilter);

  break;
}
```